src/constants.js

```diff
@@ -29,6 +29,14 @@ export const EVENT_MATCH_START = 19;
 export const EVENT_MATCH_END = 26;
 export const EVENT_VAR = 71;
 
+// Secondary Events type
+export const SHOOT = 12; // tir cadré
+export const GARDIAN_SAVE = 17; // parade
+export const GARDIAN_BLOCKED = 9999; // je croyais que c etait 14 il semble que non// arrêt ?
+export const CORNER = 16;
+export const TRANSVERSALE = 32;
+export const COUP_FRANC = 18;
```

Comment:
Non, je crois que 18 c'est faute (foul en anglais)

Reply:
Oui, une faute qui entraine un penalty c'est 72 => https://github.com/oltodo/worldcup-for-slack/issues/16

Reply:
👍